server/authentication/passport.ts

```diff
@@ -97,26 +97,34 @@ export default function passportSetup(app: Application, hmppsAuthService: HmppsA
   app.get('/sign-in', signIn)
 
   const signInCallback = (redirectUri: string) =>
+    // TODO - remove ts-ignore once keepSessionInfo has been added to types/passport
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore - No overload matches this call. - keep this until @types/passport updated
```

Comment:
`keepSessionInfo` is needed to stop req.session from being regenerated on sign in - 
when you pass in `successReturnToOrRedirect` the processing should try to redirect back to the location stored in `req.session.returnTo` on success (or `/` if returnTo is not populated)
However passport has been changed to use session.regenerate on login - this recreates the req.session object resulting in `req.session.returnTo` being dropped - so the redirect will never work correctly

a PR has been raised with definitelyTyped to update the @types/passport definition, but this could take a while to go through approval/release, so temporarily override TS error caused by unknown option